app/javascript/packs/sdk.js

```diff
@@ -62,6 +62,7 @@ const runSDK = ({ baseUrl, websiteToken }) => {
     type: getBubbleView(chatwootSettings.type),
     launcherTitle: chatwootSettings.launcherTitle || '',
     showPopoutButton: chatwootSettings.showPopoutButton || false,
+    showNotificationPopup: chatwootSettings.showNotificationPopup ?? true,
```

Comment:
Could we use a more descriptive name such as showUnreadMessagesPopup?
cc @pranavrajs @scmmishra

Reply:
showUnreadMessagesDialog

Reply:
or showUnreadMessagesNotification

Reply:
@pranavrajs @muhsin-k Done https://github.com/chatwoot/chatwoot/pull/9594/commits/b0a1fff426652c5c44f327e1a8e2f8169c870ebc